runner.py

```diff
@@ -0,0 +1,44 @@
+import pytasks
+
+
+def runner(*args):
+    """Function prints result of args taken function."""
+    if not args:
+        pytasks.main()
```

Comment:
it shouldn't find main it should run all

Reply:
solved.